bundle/language_de.properties

```diff
@@ -1,1482 +1,1493 @@
+angal.opd.editopdregistration.title                                              = Aendern registrazione DDP
+angal.operation.newoperation.title                                               = New Operation
+angal.operation.editoperation.title                                              = Edit Operation
+angal.operation.operationsbrowser.title                                          = Liste der Eingriffe
+angal.operationrowlist.newoperation.title                                        = New Operation
+angal.operationrowlist.editoperation.title                                       = Edit Operation
+angal.opetype.operationtypebrowser.title                                         = Operation Type Browser
+angal.opetype.newoperationtype.title                                             = New Operation Type
+angal.opetype.editoperationtype.title                                            = Edit Operation Type
+angal.patient.patientbrowser.title                                               = Patientenliste
+angal.patient.newpatient.title                                                   = Neuer Patient
+angal.patient.editpatient.title                                                  = Edit Patient
+angal.patient.patientselection.title                                             = Patient Selection
+angal.patientbill.newpatientbill.title                                           = New Patient Bill
+angal.patientbill.editpatientbill.fmt.title                                      = Edit Patient Bill: {0}
+angal.patvac.newpatientvaccine.title                                             = Neue patienten impfug
+angal.patvac.edipatientvaccine.title                                             = ändern patienten impfung
+angal.patvac.patientvaccinebrowser.title                                         = Patient Vaccine Browser
+angal.photoframe.photopreview.title                                              = Photo Preview
+angal.preagtreattype.pregnanttreatmenttypebrowser.title                          = Pregnant Treatment Type Browser
+angal.preagtreattype.newpregnanttreatmenttype.title                              = New Pregnant Treatment Type
+angal.preagtreattype.editpregnanttreatmenttype.title                             = Edit Pregnant Treatment Type
+angal.priceslist.listbrowser.title                                               = List Browser
+angal.priceslist.pricebrowser.title                                              = Price Browser
+angal.priceslist.newlist.title                                                   = New List
+angal.priceslist.editlist.title                                                  = Edit List
+angal.pricesothers.otherpricesbrowser.title                                      = Other Prices Browser
+angal.pricesothers.newprice.title                                                = New Price
+angal.pricesothers.editprice.title                                               = Edit Price
```

Comment:
Same doubts as in https://github.com/informatici/openhospital-gui/pull/585#discussion_r658070144